test/integration_test.js

```diff
@@ -319,6 +321,8 @@ describe("Integration tests", function() {
               expect(bucketCreate.length).eql(1);
               data = data.filter(p => !isBucketCreate(p));
             }
+            const isOwnAccount = p => p.uri == "/accounts/user";
+            data = data.filter(p => !isOwnAccount(p));
```

Comment:
Would be simpler to remove the line above and change this to:

```suggestion
            data = data.filter(p => p.uri !== "/accounts/user");
```

Also, `p` is not a very clear variable name, and so it would be preferable to use a longer and more readble one.